### PR TITLE
Fix website link http => https

### PR DESCRIPTION
**Pull request by MartinDelille**

### CODE REVIEW

**Comment by x42** [comment]
Thanks for the heads up. Checking again. This should be https://x42.github.io/libltc/  (not .com), also http still works as well.
Am I missing something?

File: README.md [hunk 1]
```diff
@@ -16,6 +16,6 @@ Documentation
 
 The API reference, examples, as well as introduction can be found at
 
-http://x42.github.com/libltc/
+https://x42.github.com/libltc/
```
Comment (MartinDelille) [inline README.md:19]:
```suggestion
https://x42.github.io/libltc/
```

**Comment by MartinDelille** [comment]
On my browser I have a 404 if I click the http link

**Comment by x42** [comment]
> On my browser I have a 404 if I click the http link

works here, also
```
curl -I https://x42.github.io/libltc/
```
shows HTTP/2 200

**Comment by x42** [comment]
Squashed and merged as 0217c50
Thanks!

**Pull request closed without merging**